XLS-0094d-dynamic-MPT

yinyiqian1 opened the pull request:
discussion link: https://github.com/XRPLF/XRPL-Standards/discussions/289

yinyiqian1 commented on XLS-0094d-dynamic-MPT/README.md:
```diff
@@ -0,0 +1,280 @@
+## 3. On-Ledger Data Structure Change
+This proposal introduces updates to the `MPTokenIssuance` ledger object, including a new optional field `MutableFlags`.
+
+## 3.1. New Optionbal Field: `MutableFlags`
```
updated: https://github.com/XRPLF/XRPL-Standards/pull/302/commits/c7efa552dfcdeded4166dd2910f352374e5dec6b

shawnxie999 commented on XLS-0094d-dynamic-MPT/README.md:
```diff
@@ -0,0 +1,280 @@
+
+| Flag Name                   | Hex Value  | Decimal Value | Description                                       |
+|-----------------------------|:----------:|:-------------:|---------------------------------------------------|
+| [Reserved]                  | ️`0x0001`   | 1             |[Reserved; used by `lsfMPTLocked`]                 |
```
maybe something like 
```suggestion
| [Reserved]                  | ️`0x0001`   | 1             |[Reserved; `lsfMPTLocked` is a flag to indicate the lock status of the token, not a configuration flag.]                 |
```

shawnxie999 reviewed (approved):
👍 looks good, left some nits

mvadari commented:
@yinyiqian1 is this ready to merge? I can merge it if you address my comment and resolve all comments.

yinyiqian1 commented:
> > > @yinyiqian1 is this ready to merge? I can merge it if you address my comment.
> > 
> > 
> > @mvadari  I just updated the spec. I added the link in `discussion-from` line, please let me know if you prefer changing it back to a raw address.
> > Other than that, the rest is ready to merge.
> 
> Raw address is preferable for now - that section is displayed with code formatting, so the link won't work.

Sure. I just changed it back.

mvadari commented on XLS-0094d-dynamic-MPT/README.md:
```diff
@@ -0,0 +1,388 @@
+  title: Dynamic Multi-Purpose Tokens
+  description: This amendment enables selected fields and flags of MPTokenIssuance to be updated after creation.
+  discussion-from: https://github.com/XRPLF/XRPL-Standards/discussions/289
+  author: Yinyi Qian <mailto:yqian@ripple.com>
```
Please also take out the `mailto` part here

mvadari commented:
@yinyiqian1 please fix the linting check and it'll be good to merge (apologies for the addition in the checks, the repo is in some amount of flux at the moment)
